prime-angular-frontend/src/app/modules/adjudication/adjudication.module.ts

```diff
@@ -76,8 +76,6 @@ import { OrganizationToaMaintenanceListPageComponent } from './pages/organizatio
 import { OrganizationToaMaintenanceViewPageComponent } from './pages/organization-toa-maintenance-view-page/organization-toa-maintenance-view-page.component';
 import { LicenseClassesMaintenancePageComponent } from './pages/license-classes-maintenance-page/license-classes-maintenance-page.component';
 import { PlrInfoComponent } from './shared/components/plr-info/plr-info.component';
-import { ContainsPipe } from '@shared/pipes/contains.pipe';
```

Comment:
These were removed by me in a different PR so that the pipe was declared in the SharedModule for other modules vs only in the AdjudicationModule.  Why is this showing up in your PR?

Reply:
Yeah I noticed that in your other PR, it doesn't exist in the adjudication.module anymore, only in the shared.module